### PR TITLE
Modify settings

### PR DESCRIPTION
**Pull request by darrenfoong**
Pull request to fix `settings.json`.

### CODE REVIEW

**Comment by darrenfoong** [comment]
I've now configured Heroku to deploy the `teddits` branch of this repo, i.e. it's the production branch.

For a start I'd like to change the title of the site from "Nova" to something else; for some reason the `title` property in `settings.json` doesn't seem to be picked up. And I've also changed some settings for the newsletter thing.

Anyway instead of directly committing to `teddits`, deploy into production, and pray that it works (ahem), I've created a PR. Heroku has this cool feature where they can deploy a separate instance of the app based on a PR. If you click "View deployment" above, you should be able to see the PR site and also realise that it's broken.

The plan is:
- work on the `df-settings` branch
- make sure everything works on the PR site
- merge `df-settings` to `teddits`
- deploy `teddits`

I'm not using `master` as the default branch, because that can be used to track the original Telescope app for updates (probably won't happen, but...). Right now I've protected `master` and `teddits` from being deleted.

Right now, all deployment is still manual, i.e. you need to go to Heroku and click around. After we get used to this, we can switch to automatic deployment.

Please comment on this PR! And help to give suggestions on how to edit `settings.json`. The [Nova documentation](http://nova-docs.telescopeapp.org/settings.html) is not great.

**Comment by darrenfoong** [comment]
TL;DR: this PR is broken and please do not merge.
